ui/app/routes/vault/cluster/access/method/item.js

```diff
@@ -0,0 +1,29 @@
+import { inject as service } from '@ember/service';
+import Route from '@ember/routing/route';
+import { singularize } from 'ember-inflector';
+
+export default Route.extend({
+  wizard: service(),
+  pathHelp: service('path-help'),
+
+  beforeModel() {
+    const { item_type: itemType } = this.paramsFor(this.routeName);
+    const { path: method } = this.paramsFor('vault.cluster.access.method');
+    let methodModel = this.modelFor('vault.cluster.access.method');
+    let { apiPath, type } = methodModel;
+    let modelType = `generated-${singularize(itemType)}-${type}`;
+    return this.pathHelp.getNewModel(modelType, method, apiPath, itemType);
```

Comment:
this is an example of how we call OpenAPI and generate a model on the auth method items pages.